docs/sources/flow/reference/components/pyroscope.scrape.md

```diff
@@ -92,35 +174,41 @@ The following blocks are supported inside the definition of `pyroscope.scrape`:
 | profiling_config > profile.fgprof             | [profile.fgprof][]             | Collect [fgprof][] profiles.                                             | no       |
 | profiling_config > profile.godeltaprof_memory | [profile.godeltaprof_memory][] | Collect [godeltaprof][] memory profiles.                                 | no       |
 | profiling_config > profile.godeltaprof_mutex  | [profile.godeltaprof_mutex][]  | Collect [godeltaprof][] mutex profiles.                                  | no       |
-| profiling_config > profile.godeltaprof_block  | [profile.godeltaprof_block][]        | Collect [godeltaprof][] block profiles.                                  | no       |
+| profiling_config > profile.godeltaprof_block  | [profile.godeltaprof_block][]  | Collect [godeltaprof][] block profiles.                                  | no       |
 | profiling_config > profile.custom             | [profile.custom][]             | Collect custom profiles.                                                 | no       |
 | clustering                                    | [clustering][]                 | Configure the component for when the Agent is running in clustered mode. | no       |
 
 The `>` symbol indicates deeper levels of nesting. For example,
 `oauth2 > tls_config` refers to a `tls_config` block defined inside
 an `oauth2` block.
 
+Any omitted blocks take on their default values. For example, 
+if `profile.mutex` is not specified in the config, 
+the defaults documented in [profile.mutex][] will be used.
+
 [basic_auth]: #basic_auth-block
 [authorization]: #authorization-block
 [oauth2]: #oauth2-block
 [tls_config]: #tls_config-block
 [profiling_config]: #profiling_config-block
-[profile.memory]: #profile.memory-block
-[profile.block]: #profile.block-block
-[profile.goroutine]: #profile.goroutine-block
-[profile.mutex]: #profile.mutex-block
-[profile.process_cpu]: #profile.process_cpu-block
-[profile.fgprof]: #profile.fgprof-block
-[profile.godeltaprof_memory]: #profile.godeltaprof_memory-block
-[profile.godeltaprof_mutex]: #profile.godeltaprof_mutex-block
-[profile.godeltaprof_block]: #profile.godeltaprof_block-block
-[profile.custom]: #profile.custom-block
+[profile.memory]: #profilememory-block
+[profile.block]: #profileblock-block
+[profile.goroutine]: #profilegoroutine-block
+[profile.mutex]: #profilemutex-block
+[profile.process_cpu]: #profileprocess_cpu-block
+[profile.fgprof]: #profilefgprof-block
+[profile.godeltaprof_memory]: #profilegodeltaprof_memory-block
+[profile.godeltaprof_mutex]: #profilegodeltaprof_mutex-block
+[profile.godeltaprof_block]: #profilegodeltaprof_block-block
+[profile.custom]: #profilecustom-block
 [pprof]: https://github.com/google/pprof/blob/main/doc/README.md
 [clustering]: #clustering-beta
 
 [fgprof]: https://github.com/felixge/fgprof
 [godeltaprof]: https://github.com/grafana/godeltaprof
 
+[delta argument]: #delta-argument
```

Comment:
> [godeltaprof]: https://github.com/grafana/godeltaprof

This is pointing to an archived repo?

Reply:
The comment is referring to https://github.com/grafana/godeltaprof
Should we change the link to: https://github.com/grafana/pyroscope-go/tree/main/godeltaprof ?

Reply:
I'll just change it